app/assets/javascripts/application.js

```diff
@@ -18,7 +18,10 @@
 //= require owl.carousel
 //= require mapbox.js
 
-$('.datetimepicker').datetimepicker();
+$('.datetimepicker').datetimepicker({
+  inline: true,
```

Comment:
Its from the datetimepicker API. I think its some javascript. I have no idea

http://xdsoft.net/jqplugins/datetimepicker/